_ml-commons-plugin/agents-tools/index.md

```diff
@@ -12,7 +12,7 @@ redirect_from:
 **Introduced 2.13**
 {: .label .label-purple }
 
-You can automate machine learning (ML) tasks using agents and tools. An _agent_ orchestrates and runs ML models and tools. A _tool_ performs a set of specific tasks. Some examples of tools are the `VectorDBTool`, which supports vector search, and the `ListIndexTool`, which executes the `cat indices` operation. For a list of supported tools, see [Tools]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/tools/index/).
+You can automate machine learning (ML) tasks using agents and tools. An _agent_ orchestrates and runs ML models and tools. A _tool_ performs a set of specific tasks. Some examples of tools are the `VectorDBTool`, which supports vector search, and the `CATIndexTool`, which executes the `cat indices` operation. For a list of supported tools, see [Tools]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/tools/index/).
```

Comment:
`CATIndexTool` should be `CatIndexTool` ?

`CatIndexTool` deleted from code in 3.0, should use `ListIndexTool` from 3.0

Reply:
Updated.